conf/modules/qc_bam.config

```
@@ -33,8 +33,10 @@ process {
         ext.prefix = { "${meta.id}_hsmetrics" }
     }
 
-    withName: '.*QC_BAM:QUALIMAP_BAMQC' {
-        ext.prefix = { "${meta.id}_qualimap" }
+    if (params.skip_qualimap) {
```

just checking, does this need to be:
```suggestion
    if (!params.skip_qualimap) {
```
as it should only run when wee are not skipping?